android/filament-android/src/main/java/com/google/android/filament/Material.java

```diff
@@ -29,45 +29,101 @@
 import java.util.List;
 import java.util.Set;
 
+/**
+ * A Filament Material defines the visual appearance of an object. Materials function as a
+ * templates from which {@link MaterialInstance}s can be spawned. Use {@link Builder} to construct
+ * a Material object.
+ */
 public class Material {
     private long mNativeObject;
     private final MaterialInstance mDefaultInstance;
 
     private Set<VertexBuffer.VertexAttribute> mRequiredAttributes;
 
+    /** Supported shading models */
     public enum Shading {
+        /** No lighting applied, emissive possible */
         UNLIT,
+
+        /** Default, standard lighting */
         LIT,
+
+        /** Subsurface lighting model */
         SUBSURFACE,
+
+        /** Cloth lighting model */
         CLOTH,
+
+        /** Legacy lighting model */
         SPECULAR_GLOSSINESS
     }
 
+    /** Attribute interpolation types in the fragment shader */
     public enum Interpolation {
+        /** Default, smooth interpolation */
         SMOOTH,
+
+        /** Flat interpolation */
         FLAT
     }
 
+    /** Supported blending modes */
     public enum BlendingMode {
+        /** Material is opaque. */
         OPAQUE,
```

Comment:
Same here and in the rest of the file really :)

Reply:
Done! FYI, do you know a way to split an anchor tag into multiple lines? I can't find a way to fit the 100 character count, but it may just be something we have to live with.